### PR TITLE
Popup feature

### PR DESCRIPTION
**Pull request by Graciano1997**
In this project @Graciano1997  and @Ayokunnumi1  did:

- Created a mobile and window popup to show the projects
- Update readme file
- Changed the deployed branch to popup feature
- Fix linter issues

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+const seeProjectList = document.querySelectorAll('.btnPopup');
+window.addEventListener('resize', handleResizeMobile);
+
+seeProjectList.forEach((seeProject) => {
+  seeProject.addEventListener('click', () => {
+    if (windowSizemobile < 768) {
+      createPopup(project);
+    }
+  });
+});
```
Comment (usorfaitheloho) [inline assets/js/main.js:209]: You did well with your popup modal and you have a beautiful design, weldone @Graciano1997 👍 

Please implement the following changes, as instructed in the project requirement you are to make your design to be similar to the design given. While you tried in achieving that, you have some corrections to make :

1. In the desktop view, of your popup modal , you are having an issue with height and alignment of the popup which doesn't give you a good UI for your portfolio. 

Your design :


https://github.com/Graciano1997/portofolio-graciano/assets/20537712/4d88c005-11bc-4d67-b1a2-2c10d4a01b93


2. For the desktop view, add a margin at the bottom , so your text isn't cut out as it appears in your design 

Your design :
image

3. For your mobile view, kindly use the right font-color for all your text and fix the upper part of the mobile version to be similar to our figmma design 👍 

Figma design :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Graciano1997 ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI design ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/main.js (hunk 1 above)
Comment (Graciano1997) [reply assets/js/main.js]: sorry but it is taking some additional seconds to be rendered on the screen

**Review by skyv26 — APPROVED** [review]
## Hi @Graciano1997, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Graciano1997** [comment]
233992044-ad00b20f-3f2f-4c86-8b94-1e613432af5f

**Pull request merged**